src/services/chat_service.py

```
@@ -0,0 +1,154 @@
+
+    print(f"🌍 국가 필터링 적용: ID {country_id}")
+
+    # 1. [임베딩] 질문을 벡터로 변환
+    try:
+        text_input = TextEmbeddingInput(text=query, task_type="RETRIEVAL_QUERY")
+        embeddings = embedding_model.get_embeddings([text_input])
+        query_vector = embeddings[0].values
+    except Exception as e:
+        print(f"❌ 임베딩 실패: {e}")
+        # Vertex AI 통신 오류가 여기서 발생할 가능성이 높음
+        raise e
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)